components/bl-list-component/src/index.js

```diff
@@ -0,0 +1,34 @@
+import { useRef } from "react";
+
+import { List } from "./list";
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function ListComponent({ component, eventHandlers }) {
+  const { classList, width, backgroundColor, type, listItems, color, fontSize, padding } = component;
```

Comment:
Here is type, but in item signature is typeOfList - that should be the same. Also, don;t forget to change XML example.

Reply:
It's a correct names. 'type' is a global type of a list, if we didn't have a type as 'typeOfList' parameter for a child, we get it as a global type.

Reply:
That not answering why we have two entities such as type and typeOfList that do absolutely the same thing.